src/SignalR/server/SignalR/test/HubConnectionHandlerTests.cs

```diff
@@ -2911,6 +2911,37 @@ bool ExpectedErrors(WriteContext writeContext)
             }
         }
 
+        [Fact]
+        public async Task StreamUploadBufferCapacityBlocksOtherInvocations()
+        {
+            var serviceProvider = HubConnectionHandlerTestUtils.CreateServiceProvider(services =>
+            {
+                services.Configure<HubOptions>(options =>
+                {
+                    options.StreamBufferCapacity = 1;
```

Comment:
Wouldn't this test still pass with a StreamBufferCapacity of 5 since you send 5 items? I think the capacity and items sent should match in this test to better demonstrate the relationship.

Reply:
4, but yeah. I agree